src/pages/Autolending/RiskRatingDropdown.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<div>
+		<h3 class="filter-title">
+			Risk Rating
+		</h3>
+		<kv-dropdown-rounded v-model="riskRating">
+			<option value="1">
+				&#9733; and up
+			</option>
+			<option value="2">
+				&#9733; &#9733; and up
+			</option>
+			<option value="3">
+				&#9733; &#9733; &#9733; and up
+			</option>
+			<option value="4">
+				&#9733; &#9733; &#9733; &#9733; and up
+			</option>
+		</kv-dropdown-rounded>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import gql from 'graphql-tag';
+import KvDropdownRounded from '@/components/Kv/KvDropdownRounded';
+
+export default {
+	inject: ['apollo'],
+	components: {
+		KvDropdownRounded,
+	},
+	data() {
+		return {
+			riskRating: '1',
```

Comment:
Use a number `1` instead of a string `'1'`, because the value fetched in the query will be a number